### PR TITLE
Feature/base structure

### PR DESCRIPTION
**Pull request by AlexeyUlyanov96**
Create base structure app, delete manifect.json and serviceWorker, because we aren't need it now.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import './App.css';
+import NavBar from "./components/NavBar";
+import MainPage from "./components/MainPage";
+
+function App() {
+  return (
```
[the remark this answers is not in the document]
Comment (AlexeyUlyanov96) [reply src/App.js]: do you mean line break?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "my-app",
```
[the remark this answers is not in the document]
Comment (AlexeyUlyanov96) [reply package.json]: Fixed. Renamed by fitness

File: public/index.html [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app"
+    />
+    <link rel="apple-touch-icon" href="logo192.png" />
+    <title>React App</title>
```
[the remark this answers is not in the document]
Comment (AlexeyUlyanov96) [reply public/index.html]: Fixed. Renamed by fitness

File: src/App.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import './App.css';
+import NavBar from "./components/NavBar";
+import MainPage from "./components/MainPage";
```
[the remark this answers is not in the document]
Comment (AlexeyUlyanov96) [reply src/App.js]: Fixed. Renamed by HomePage

File: src/App.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (AlexeyUlyanov96) [reply src/App.js]: Fixed. Used arrow function, delete braces and return method

File: src/components/MainPage.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const MainPage = () => {
+    return (
```
[the remark this answers is not in the document]
Comment (AlexeyUlyanov96) [reply src/components/MainPage.js]: Fixed. Used arrow function, delete braces and return method

File: src/components/NavBar.js [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+            </div>
+            <div>
+                Profile
+            </div>
+            <div>
+                Categories
+            </div>
+        </div>
+    )
+};
```
[the remark this answers is not in the document]
Comment (AlexeyUlyanov96) [reply src/components/NavBar.js]: Fixed. Used arrow function, delete braces and return method

File: src/index.css [hunk 7]
```diff
@@ -0,0 +1,13 @@
+body {
+  margin: 0;
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
+    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
+    sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+code {
```
[the remark this answers is not in the document]
Comment (AlexeyUlyanov96) [reply src/index.css]: Fixed.

File: src/index.js [hunk 8]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+
+ReactDOM.render(<App />, document.getElementById('root'));
```
[the remark this answers is not in the document]
Comment (AlexeyUlyanov96) [reply src/index.js]: used normalize.css

**Comment by avmaxim** [comment]
@AlexeyUlyanov96 looks nice!

**Pull request merged**